tcms/testplans/templates/testplans/mutable.html

```diff
@@ -92,7 +112,27 @@
                 </div>
             </div>
 
+            {% if test_plan %}
+                <div class="form-group">
+                    <label class="col-md-2 col-lg-2">{% trans "Notification Include" %}</label>
+                    <div class="col-md-10 col-lg-10">
+                        <div>{{ form.auto_to_plan_author }} {{ form.auto_to_plan_author.label }} {{ form.auto_to_plan_author.errors }}</div>
+                        <div>{{ form.auto_to_case_owner }} {{ form.auto_to_case_owner.label }} {{ form.auto_to_case_owner.errors }}</div>
+                        <div>{{ form.auto_to_case_default_tester }} {{ form.auto_to_case_default_tester.label }} {{ form.auto_to_case_default_tester.errors }}</div>
+                    </div>
+
+                    <label class="col-md-2 col-lg-2">{% trans "Notification Trigger" %}</label>
+                    <div class="col-md-10 col-lg-10">
+                        <div>{{ form.notify_on_plan_update }} {{ form.notify_on_plan_update.label }} {{ form.notify_on_plan_update.errors }}</div>
+                        <div>{{ form.notify_on_case_update }} {{ form.notify_on_case_update.label }} {{ form.notify_on_case_update.errors }}</div>
+                    </div>
+                </div>
+            {% endif %}
+
             <button type="submit" class="btn btn-default btn-lg">{% trans "Save" %}</button>
+            {% if test_plan %}
+                <button type="reset" class="btn btn-warning btn-lg" value="Reset"> {% trans "Reset" %}</button>
```

Comment:
I am still not convinced this extra button gives us anything useful. My vote is to remove it unless you can find compelling evidence that having a button that will reset the form without reloading the page is something that modern web needs.